universal-login-react/test/test.tsx

```diff
@@ -0,0 +1,15 @@
+import 'jsdom-global/register';
+import React from 'react';
+import {expect} from 'chai';
+import {TestDiv} from '../src/test';
+import {configure, ReactWrapper, mount} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({adapter: new Adapter()});
```

Comment:
Why not do it like in [wallet](https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-wallet/test/integration/helpers/configuration.ts) (as a configuration file, that we require in test script)?

Reply:
👍 To do in next PR